src/vivarium/framework/results/manager.py

```diff
@@ -71,6 +72,8 @@ def get_results(self) -> Dict[str, pd.DataFrame]:
 
     # noinspection PyAttributeOutsideInit
     def setup(self, builder: "Builder") -> None:
+        self._results_context.setup(builder)
```

Comment:
Was this previously not being called at all?

Reply:
It wasn't, though there previously wasn't anything interesting there (only the logger). But now there's the excluded categories